### PR TITLE
Default profile picture for Chorus

### PR DESCRIPTION
**Pull request by AndrewLngdn**
Adds default profile pictures to new users based on first and last initial

### CODE REVIEW

**Comment by AndrewLngdn** [comment]
Green build!!

**Comment by kevin-alpine** [comment]
Woot!

File: components/api/app/controllers/api/user_images_controller.rb [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # KT TODO: does this file work, since the ImagesController is broken?
 module Api
   class UserImagesController < ImagesController
+
+    def destroy
+      UserImageGenerator.create_profile_image_for(@entity)
+      authorize! :update, @user, current_user, nil
+      present @entity.image, :content_type => 'text/html'
```
Comment (mmilano) [inline components/api/app/controllers/api/user_images_controller.rb]: is this the content type for the svg file?

Comment (AndrewLngdn) [inline components/api/app/controllers/api/user_images_controller.rb]: I copied the content type of https://github.com/Chorus/chorus/blob/master/components/api/app/controllers/api/images_controller.rb#L9, although it looks like it should actually be 'text/json'. SVG is never sent over the wire, it only resides in the backend.

**Pull request merged**